### PR TITLE
SL IC solution advriskmin1

### PR DESCRIPTION
**Pull request by patriciomassaro**
- Adding Solution for SL IC exercise 1

### CODE REVIEW

**Comment by patriciomassaro** [comment]
[IC solutions SL](https://trello.com/c/62sCOyAY/219-ic-solutions-sl)

File: exercises/advriskmin/ex_rnw/sol_ic_advriskmin_1.Rnw [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      &= \E_{xy} \left[ (y - a)^2 \right]\\
+      &= \E_y [ y^2 ] - 2a \underbrace{\E_y[y]}_{a} + a^2 \\
+      &= \E_y [ y^2 ] - a^2 
+    \end{align*}
+    
+    As $\E_y[y^2]$ is yet unknown, the calculation is detailed below: 
+    \lz
+    \begin{align*} 
+      \E_y [y^2] &= \E_x \left[ \E_{y_{|x}} \left[ y^2_{|x} \right] \right] \\
+      &= \E_x \left[ \var_{y_{|x}} [y_{|x}] + \E_{y_{|x}} \left[ y_{|x} \right] \right] \\
```
Comment (ludwigbothmann) [inline exercises/advriskmin/ex_rnw/sol_ic_advriskmin_1.Rnw:71]: I guess, here is a square missing at the end (squaring the E?)

**Review by ludwigbothmann — CHANGES_REQUESTED** [review]
Could you rename the files, please, @patriciomassaro ? It should be "ic_sol_...". Thank you

File: exercises/advriskmin/ex_rnw/sol_ic_advriskmin_1.Rnw (hunk 1 above)
Comment (patriciomassaro) [reply exercises/advriskmin/ex_rnw/sol_ic_advriskmin_1.Rnw]: Indeed! I changed it in the next commit

Comment (patriciomassaro) [reply exercises/advriskmin/ex_rnw/sol_ic_advriskmin_1.Rnw]: > Could you rename the files, please, @patriciomassaro ? It should be "ic_sol_...". Thank you

Done

**Comment by lisa-wm** [comment]
@ludwigbothmann, this should be migrated to the new lecture_sl repo

**Comment by patriciomassaro** [comment]
[IC solutions SL](https://trello.com/c/62sCOyAY/219-ic-solutions-sl)

**Pull request closed without merging**